Parse/Parse/Resources/Parse-iOS.Info.plist

```diff
@@ -23,6 +23,6 @@
 	<key>CFBundleVersion</key>
 	<string>2.7.3</string>
 	<key>MinimumOSVersion</key>
-	<string>9.0</string>
+	<string>12.0</string>
```

Comment:
Is this bumping the required min iOS  version of the SDK, ergo a breaking change?

Reply:
The minimum version is 12. Check `package.swift`. 9.0 is wrong